packages/types/src/DidHelpers.ts

```diff
@@ -94,7 +92,7 @@ type Base58Btc = string
 /** Multibase encoding of a public- or private key including multicodec variant flag. */
 export type KeyMultibaseEncoded = `z${Base58Btc}`
 
-export type AcceptedSigners =
+export type DidHelpersAcceptedSigners =
```

Comment:
Why do we need the `DidHelpers` prefix here? maybe we can also use it outside the helpers, for example in holder options?

Reply:
That makes sense. Any reason why you'd not use it Raph?

Reply:
I added the prefix when moving it out of the DidHelpers because it would otherwise a suggest a generality that may not be entirely accurate. The DidHelpers accept these signers (and others might, too), but not everything does afaik.